edb/server/compiler/compiler.py

```diff
@@ -2139,6 +2139,11 @@ async def describe_database_restore(
         )
         ctx.state.start_tx()
 
+        dump_with_extraneous_computables = (
+            dump_server_ver is not None
+            and dump_server_ver < (1, 0, verutils.VersionStage.ALPHA, 8)
```

Comment:
Is it still fine to use on patched version of edgedb?

Reply:
Yes. The backpatched version only takes care of the `multi` computable case, leaving the old behavior of dumping `single` computable columns in place.